README.md

```diff
@@ -26,12 +26,12 @@ func main() {
 	clientSecret := os.Getenv("CLIENT_SECRET")
 	baseURL := os.Getenv("BASE_URL")
 
-	client := elation.NewClient(httpClient, tokenURL, clientID, clientSecret, baseURL)
+	client := elation.NewHttpClient(httpClient, tokenURL, clientID, clientSecret, baseURL)
```

Comment:
This should be `NewHTTPClient`

Reply:
I don't think that's the case: https://github.com/authorhealth/go-elation/blob/main/client.go#L88

Or are you requesting that this function name (and presumably also struct name) be changed?

Reply:
I've made the assumption you wanted the rename, which I have done.